beps/0001-notifications-system/README.md

```diff
@@ -85,12 +85,43 @@ The role of the notifications plugin is to manage the lifecycle of notifications
 The notification backend stores notification using the [database service](https://backstage.io/docs/backend-system/core-services/index#database). In particular it needs to store the following information for each notification:
 
 - ID
-- Payload
+- Title
```

Comment:
Wondering if the title should be part of the payload instead?

Reply:
I see a notification to be always human-readable, the target consumer is primarily a person. It means we are not aiming at sort of inter-process messaging system.

By considering that, the title is a mandatory field. In fact, it is "the main reason" for creating the message. With this logic, I would keep it among the top-level fields. It's sort of identification of the object from the end-user perspective.

I consider the "payload" as a mix of both optional and mandatory fields which are further enriching the message for next processing. The "title" is different.